frontend/src/features/projects/common/projectConverter.ts

```diff
@@ -103,9 +103,9 @@ const toApiProperty = (property: IProperty): IApiProperty => {
       postal: property.postal,
       provinceId: property.province,
     },
-    zoning: property.zoning,
-    zoningPotential: property.zoningPotential,
-    municipality: property.municipality,
+    zoning: property.zoning!,
```

Comment:
hmm, what is the rationale for this and the above change?

Reply:
this was something i was going to ask about, i was getting ts errors here after my changes. would it be better to have it as `property.x ?? ''`